specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -26,6 +26,11 @@ useful for a wide audience.
 ## Glossary
 
 * VDK: https://github.com/vmware/versatile-data-kit/wiki/dictionary#vdk
+* Notebook: A .ipynb file.
+* Notebook Step:  A single unit of work for a Data Job that includes notebook files. A SQL step is a SQL statement. A python step is the
```

Comment:
Glossary is suppose to contain terms that are defined before hand - e.g. WHat is a a 'notebook' is appropriately here. 

You are introducing VDK Notebook step as part of this VEP. Before that it did not exists as a term. It needs to go into the Design part of the document.